doc/user_manual/rom.tex

```diff
@@ -2833,6 +2833,9 @@ \subsubsection{ARMA}
   \default{Time}
   \item \xmlNode{Features}, \xmlDesc{string, required field}, defines the features (e.g., scaling). Note that only one feature is allowed for \xmlString{ARMA} and in current implementation this is used for evaluation only.
   \item \xmlNode{Target}, \xmlDesc{string, required field}, defines the variables of the time series.
+  \item \xmlNode{reseedCopies}, \xmlDesc{boolean, optional field}, if True then whenever the ARMA is copied, a
+    random reseeding will be performed to ensure different histories.
+  \default{True}
```

Comment:
It looks to me like reseedCopies is mentioned in the manual here.  Is there another place you'd like me to mention it?